app/models/katello/glue/pulp/pulp_content_unit.rb

```diff
@@ -36,7 +36,7 @@ def in_repositories(repos)
       end
 
       def pulp_data(uuid)
-        unit_handler.find_by(:unit_id => uuid)
+        unit_handler.find_by_unit_id(uuid)
```

Comment:
I gotcha. So we don't want to switch to ActiveRecords new `find_by`. Got it.

Would it be worth looking into changing the syntax of Runcible at some point down the road? Or just leave it?